### PR TITLE
Fixed typos and added a reference.

### PR DESCRIPTION
**Pull request by andrew-mackie**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Review by thecodingaviator — APPROVED** [review]
LGTM!

File: guide/english/computer-science/data-structures/graphs/index.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Graphs
 ---
 ## Graphs
-A graph is a data-structure you can use to solve routing problems, such as "Are these two components connected?" and "What is the shortest path from point a to b?"
+A graph is a data structure you can use to solve routing problems, such as "Are these two components connected?" and "What is the shortest path from point a to point b?"
```
Comment (co16353sidak) [inline guide/english/computer-science/data-structures/graphs/index.md]: As a suggestion, we don't know about the points a or b previously in this guide article, so to increase readability we could change the line `from point a to point b` to `from a point a to a point b` which makes the user understand that we are talking about any two points here.

**Review by theNaut1lus — CHANGES_REQUESTED** [review]
@andrew-mackie just a small grammatical suggestion for you, otherwise LGTM.

/cc: @RandellDawson

File: guide/english/computer-science/data-structures/graphs/index.md (hunk 1 above)
Comment (RandellDawson) [reply guide/english/computer-science/data-structures/graphs/index.md]: @co16353sidak I believe your suggestion of adding two extra `a`s makes it even more confusing.  I think it is fine as is for now.

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**